tomcat/src/main/scala/org/http4s/server/tomcat/TomcatBuilder.scala

```diff
@@ -120,6 +121,9 @@ sealed class TomcatBuilder[F[_]] private (
     })
 
   def mountService(service: HttpRoutes[F], prefix: String): Self =
+    mountServiceHttpApp(http4s.httpRoutesToApp(service), prefix)
+
+  def mountServiceHttpApp(service: HttpApp[F], prefix: String): Self =
```

Comment:
I think I would just call this `mountHttpApp`.

On blaze, it's called `witHttpApp`, but the servlet model is a little different: we have more than one.  So I think the different name is justified.